[material-components] Issues with new 0.12.1 update

Hey @r0man, 

I'm having trouble with 0.12.1.  I'm guessing it has to do with:
https://github.com/material-components/material-components-web/commit/e21ec90
and possibly with:
https://github.com/material-components/material-components-web/issues/767

Do the externs need to be updated for mdc-base?  Do I need to refer to mdc-base now instead of mdc?  What is the way it should be used?

This is my traceback when I change from 0.10.0 to 0.12.1:
```javascript
material-components.inc.js:90 Uncaught Error: Cannot find module "@material/base"
    at Object.<anonymous> (material-components.inc.js:90)
    at __webpack_require__ (material-components.inc.js:35)
    at Object.defineProperty.value (material-components.inc.js:81)
    at material-components.inc.js:84
    at webpackUniversalModuleDefinition (material-components.inc.js:14)
    at material-components.inc.js:15
(anonymous)	@	material-components.inc.js:90
__webpack_require__	@	material-components.inc.js:35
Object.defineProperty.value	@	material-components.inc.js:81
(anonymous)	@	material-components.inc.js:84
webpackUniversalModuleDefinition	@	material-components.inc.js:14
(anonymous)	@	material-components.inc.js:15
```

and

```javascript
Uncaught ReferenceError: mdc is not defined
    at Object.<anonymous> (material-components.cljs:79)
    at Object.reagent$impl$component$custom_wrapper_$_componentDidMount [as componentDidMount] (component.cljs:188)
    at react-dom.inc.js:4704
    at measureLifeCyclePerf (react-dom.inc.js:4514)
    at react-dom.inc.js:4703
    at CallbackQueue.notifyAll (react-dom.inc.js:948)
    at ReactReconcileTransaction.close (react-dom.inc.js:11361)
    at ReactReconcileTransaction.closeAll (react-dom.inc.js:14768)
    at ReactReconcileTransaction.perform (react-dom.inc.js:14715)
    at ReactUpdatesFlushTransaction.perform (react-dom.inc.js:14702)
(anonymous) @ material-components.cljs:79
mdc-textfield_componentDidMount @ component.cljs:188
(anonymous) @ react-dom.inc.js:4704
measureLifeCyclePerf @ react-dom.inc.js:4514
(anonymous) @ react-dom.inc.js:4703
notifyAll @ react-dom.inc.js:948
close @ react-dom.inc.js:11361
closeAll @ react-dom.inc.js:14768
perform @ react-dom.inc.js:14715
perform @ react-dom.inc.js:14702
perform @ react-dom.inc.js:12807
flushBatchedUpdates @ react-dom.inc.js:12890
closeAll @ react-dom.inc.js:14768
perform @ react-dom.inc.js:14715
batchedUpdates @ react-dom.inc.js:8760
enqueueUpdate @ react-dom.inc.js:12918
enqueueUpdate @ react-dom.inc.js:12516
enqueueForceUpdate @ react-dom.inc.js:12649
ReactComponent.forceUpdate @ react.inc.js:1268
reagent$impl$batching$run_queue @ batching.cljs:39
reagent.impl.batching.RenderQueue.flush_queues @ batching.cljs:86
reagent.impl.batching.RenderQueue.run_queues @ batching.cljs:76
(anonymous) @ batching.cljs:63
```

I'm including this with:
```cljs
            [cljsjs.material-components :as mdc]
```

CI setup uses Node 7.10: https://github.com/cljsjs/packages/blob/master/Dockerfile

Logs are here: https://circleci.com/gh/cljsjs/packages/2125 and they show lots of problems like this:

```
    ERROR in ./packages/mdc-dialog/index.js
    Module not found: Error: Can't resolve '@material/base' in '/root/.boot/cache/tmp/root/cljsjs/material-components/3xn/g0mg04/material-components-web-0.12.1/packages/mdc-dialog'
     @ ./packages/mdc-dialog/index.js 27:0-46
     @ multi ./packages/mdc-dialog/index.js
```

--- a/material-components/build.boot
+++ b/material-components/build.boot
@@ -6,9 +6,9 @@
          '[boot.core :as boot]
          '[boot.tmpdir :as tmpd]
          '[clojure.java.io :as io]
-         '[boot.util :refer [sh]])
+         '[boot.util :refer [dosh]])
 
-(def +lib-version+ "0.12.1")
+(def +lib-version+ "0.13.0")
 (def +version+ (str +lib-version+ "-0"))
 
 (task-options!
@@ -29,15 +29,15 @@
         (io/make-parents target)
         (io/copy (tmpd/file f) target))
       (binding [boot.util/*sh-dir* (str (io/file tmp (format "material-components-web-%s" +lib-version+)))]
-        ((sh "npm" "install"))
-        ((sh "npm" "run" "dist")))
+        (dosh "npm" "install")
+        (dosh "npm" "run" "dist"))
       (-> fileset (boot/add-resource tmp) boot/commit!))))
 
 (deftask package []
   (task-options! push {:ensure-branch nil})
   (comp
    (download :url (str "https://github.com/material-components/material-components-web/archive/v" +lib-version+ ".zip")
-             :checksum "036cae2737b5635211b7a763f2fc8c1c"
+             :checksum "b0a844d7b19a5e936a1628c43c9fb1a4"
              :unzip true)
 
    (build-material-components)

--- a/material-components/build.boot
+++ b/material-components/build.boot
@@ -30,6 +30,7 @@
         (io/copy (tmpd/file f) target))
       (binding [boot.util/*sh-dir* (str (io/file tmp (format "material-components-web-%s" +lib-version+)))]
         (dosh "npm" "install")
+        (dosh "npm" "run" "postinstall")
         (dosh "npm" "run" "dist"))
       (-> fileset (boot/add-resource tmp) boot/commit!))))
 

